CPAC/anat_preproc/anat_preproc.py

```diff
@@ -130,13 +151,40 @@ def create_anat_preproc(method='afni', already_skullstripped=False, config=None,
     else:
         preproc.connect(anat_deoblique, 'out_file', anat_reorient, 'in_file')
 
+    # TODO reassess average center of mass implementation, esp for skull alignment
+    '''
+    if 'anat' in config.run_longitudinal:
+        anat_align_cmass = pe.Node(interface=afni.CenterMass(), name='cmass')
```

Comment:
I got 

```Python
200711-19:44:42,832 nipype.workflow INFO:
   [Node] Running "template_skull_for_anat_center_of_mass" ("nipype.interfaces.afni.utils.CenterMass"), a CommandLine Interface with command:
3dCM /media/ebs/runs/2020-07-11_feature-longitudinal/working/anat_longitudinal_template_0027234/template_skull_for_anat_center_of_mass/MNI152_T1_2mm.nii.gz > template_center_of_mass.txt
200711-19:44:42,834 nipype.workflow WARNING:
   [Node] Error on "anat_longitudinal_template_0027234.template_skull_for_anat_center_of_mass" (/media/ebs/runs/2020-07-11_feature-longitudinal/working/anat_longitudinal_template_0027234/template_skull_for_anat_center_of_mass)
200711-19:44:42,835 nipype.workflow ERROR:
   Node template_skull_for_anat_center_of_mass failed to run on host 8a09b9689bb3.
200711-19:44:42,835 nipype.workflow ERROR:
   Saving crash info to /media/ebs/runs/2020-07-11_feature-longitudinal/crash/crash-20200711-194442-UID1000-template_skull_for_anat_center_of_mass-1cce622e-38d6-4917-8237-d53d7a5f3461.pklz
Traceback (most recent call last):
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/plugins/linear.py", line 44, in run
  node.run(updatehash=updatehash)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 471, in run
  result = self._run_interface(execute=True)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 555, in _run_interface
  return self._run_command(execute)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/pipeline/engine/nodes.py", line 635, in _run_command
  result = self._interface.run(cwd=outdir)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 521, in run
  runtime = self._run_interface(runtime)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/afni/base.py", line 110, in _run_interface
  return super(AFNICommandBase, self)._run_interface(runtime)
File "/usr/local/miniconda/lib/python3.7/site-packages/nipype/interfaces/base/core.py", line 1024, in _run_interface
  runtime.hostname))
OSError: No command "3dCM" found on host 8a09b9689bb3. Please check that the corresponding package is installed.
```

https://github.com/FCP-INDI/C-PAC/pull/1320 fixed the issue for me